parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs

```diff
@@ -52,5 +62,166 @@ bridge_hub_test_utils::test_cases::include_teleports_for_native_asset_works!(
 			_ => None,
 		}
 	}),
-	1013
+	bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID
+);
+
+bridge_hub_test_utils::include_initialize_bridge_by_governance_works!(
+	initialize_bridge_to_wococo_by_governance_works,
+	Runtime,
+	BridgeGrandpaWococoInstance,
+	bridge_hub_test_utils::CollatorSessionKeys::new(
+		AccountId::from(ALICE),
+		AccountId::from(ALICE),
+		SessionKeys { aura: AuraId::from(sp_core::sr25519::Public::from_raw(ALICE)) }
+	),
+	bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+	Box::new(|call| RuntimeCall::BridgeWococoGrandpa(call).encode())
+);
+
+bridge_hub_test_utils::include_handle_export_message_from_system_parachain_to_outbound_queue_works!(
+	handle_export_message_from_system_parachain_to_outbound_queue_works_for_wococo,
```

Comment:
I think this should be named
```suggestion
	handle_export_message_from_system_parachain_to_outbound_queue_works_for_rococo,
```
since it tests that BHR handles `ExportMessage` and adds it to outbound queue (towards BHW)

or if it's still confusing, maybe:

```suggestion
	rococo_handle_export_message_from_system_parachain_add_to_outbound_queue_for_wococo_works,
```

Reply:
ok, got your point,
if we put rococo tests also behind `mod bridge_hub_rococo` as you suggested bellow,
then we can have just the same name `handle_export_message_from_system_parachain_to_outbound_queue_works` for both